app/routes/search/suggestions.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, ResponseToolkit, ResponseObject } from "@hapi/hapi";
+import Joi from "joi";
+import { getSearchSuggestions } from "../../backend/api";
+
+module.exports = [
+  {
+    path: '/suggestions',
+    method: 'GET',
+    options: {
+      auth: false,
+      validate: {
+        query: Joi.object({
+          searchString: Joi.string().regex(/^[a-zA-Z0-9 '-]*$/).trim().min(1).required(),
```

Comment:
As discussed, this is not part of the ticket but since allowing certain special chars is causing server crashes I have asked Gavin to decide on whether we want to leave this for now and address it after the spike to decide on allowed special chars.

Reply:
Sid, as discussed earlier, you have agreed to keep this in this story with Gavin. based on that we can continue with this.